examples/spark2/src/main/scala/org/apache/carbondata/examples/DataFrameComplexTypeExample.scala

```diff
@@ -56,6 +72,38 @@ object DataFrameComplexTypeExample {
          | 'dictionary_include'='city')
          | """.stripMargin)
 
+    spark.sql(
+      s"""
+         | CREATE TABLE ${ complexTypeNoDictionaryTableNameArray }(
+         | id INT,
+         | name STRING,
+         | city STRING,
+         | salary FLOAT,
+         | file array<string>
+         | )
+         | STORED BY 'carbondata'
+         | TBLPROPERTIES(
+         | 'sort_columns'='name',
+         | 'dictionary_include'='city')
+         | """.stripMargin)
+
+
+    spark.sql(
+      s"""
+         | CREATE TABLE ${ complexTypeNoDictionaryTableName }(
+         | id INT,
+         | name STRING,
+         | city STRING,
+         | salary FLOAT,
+         | file struct<school:array<string>, school1:array<string>, age:int>
+         | )
+         | STORED BY 'carbondata'
+         | TBLPROPERTIES(
+         | 'sort_columns'='name',
+         | 'dictionary_exclude'='val')
```

Comment:
Need to fix this example class after PR#2266 is merged. It will make complex columns as dictionary_exclude by default.